lib/rules/property-no-unknown/__tests__/index.js

```diff
@@ -360,3 +360,56 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
+	config: [true, { ignoreAtRules: ['media', '/^my-/', /^custom-/i] }],
+
+	accept: [
+		{
+			code: 'a { @media { foo: 1 } }',
```

Comment:
Let's use standard CSS in all the tests, i.e. only nesting rule-sets within at-rules (and not the other way around):

For example:

```css
@supports(display: grid) {
  .foo {
    my-property: 1;
  }
}
```

I suspect we'll find that this test will fail because only the parent node is being checked.